templates/versioning/latest_releases.html

```diff
@@ -6,16 +6,16 @@
 
 {% block content %}
 
-<h1> Coding System Releases</h1>
+  <h1> Coding System Releases</h1>
 
-{% for coding_system_release in latest_releases %}
+  {% for coding_system_release in latest_releases %}
     <h4>{{ coding_system_release.name }}</h4>
     <ul>
-        <li>Latest release: {{ coding_system_release.release_name }}</li>
-        <li>Valid from: {{ coding_system_release.release.valid_from }}
+      <li>Latest release: {{ coding_system_release.release_name }}</li>
+      <li>Valid from: {{ coding_system_release.release.valid_from }}
         <li>Imported on: {{ coding_system_release.release.import_timestamp }}
-    </ul>
+        </ul>
 
-{% endfor %}
+  {% endfor %}
```

Comment:
I realise this is a different issue, but the odd indenting here made me realise that this doc is missing some closing `</li>` tags